scripts/apps/search/controllers/SearchMenuController.ts

```diff
@@ -56,7 +57,7 @@ export default function SearchMenuController(
 
     const initActiveProvider = () => {
         if ($location.path() === '/search') {
-            if (!$location.search()._id) {
+            if (!$location.search()._id && !$location.search().item) {
```

Comment:
Can you add a comment in the code describing what this line does and when is it relevant?